cli/genkey/genkey_test.go

```diff
@@ -1 +1,49 @@
 package genkey
+
+import (
+	"encoding/json"
+	"github.com/cloudflare/cfssl/cli"
```

Comment:
One last thing. Put this import in a separated block. Examples can be found in the code base.